### DISCUSSION TITLE
Celery/Redis priority doesn't work with KEDA scaling

### DISCUSSION THREAD

**Discussion by Alekky09** [celery/kombu#1769] [Q&A]
I've been trying to use Celery with Redis backend for my app with KEDA scaling based on the Redis list length.
So far what I've gotten out of it is:
- KEDA doesn't see lists because of the \x06\x16 separator characters in the priority queues;
- There's a bug with changing the separator characters which causes the workers to go in a CrashLoop:
```
File "/usr/local/lib/python3.8/site-packages/kombu/transport/virtual/exchange.py", line 62, in <setcomp>
queue for rkey, _, queue in table
ValueError: not enough values to unpack (expected 3, got 1)
```
- Cleaning the queue and restarting the workers doesn't help;

**Comment by agronick** [comment]
Is there any way around this? Seems like a pretty big bug. Looks like every control command throws an exception if you are using queues with a custom separator. 

It seems like just running flushdb on redis can help with that second issue.

**Reply by Alekky09** [reply]
I never found a way around it unfortunately
